src/components/layout/Footer.vue

```diff
@@ -1,116 +1,202 @@
 <template>
-    <nav
-        class="h-12 w-full overflow-hidden fixed bottom-0 left-0 z-20 text-white flex flex-col lg:flex-row border-t bd-main bg-main"
-    >
-        <div class="invisible md:visible">
-            <div class="flex overflow-hidden absolute w-64 h-12 border-r bd-main">
-                <div class="flex px-4 py-4 cursor-pointer">
-                    <img src="@/assets/images/green-dot.svg" class="h-full py-0.5">
-                    <span class="px-1 text-xs my-auto font-normal px-3">13224549</span>
-                    <img src="@/assets/images/external-link.svg" >
-                </div>
-            </div>
-            <div class="flex absolute w-96 mr-4 right-0 p-2 text-right h-12 lg:border-l bd-main">
-                <div class="flex absolute right-2">
-                    <img src="@/assets/images/socials/twitter.svg" class="mx-2 my-auto cursor-pointer">
-                    <img src="@/assets/images/socials/discord.svg" class="mx-2 my-auto cursor-pointer">
-                </div>
-            </div>
+  <nav
```

Comment:
why are you formatting other files that you shouldnt be touching on this branch?